layouts/partials/head.html

```diff
@@ -27,12 +28,14 @@
     <meta property="og:image" content="{{ absURL $image }}">
     <meta property="og:image:secure_url" content="{{ absURL $image }}">
 
+    {{ if $hasTwitter }}
```

Comment:
Why do we want this change ? The twitter meta tags decide how the post should be displayed on Twitter.
The user does not have twitter social icon does not matter

Ref: https://www.digitalocean.com/community/tutorials/how-to-add-twitter-card-and-open-graph-social-metadata-to-your-webpage-with-html

Reply:
Got that, I will revert this change.